app/scripts/directives/istagSelect.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
-angular.module("openshiftConsole")
+(function() {
```

Comment:
As we convert these, we should move them to the new directory structure.

Reply:
makes sense ! wasnt sure since we still have most of the components in the directives folder. Will move it in this PR